code/game/objects/items/weapons/dice.dm

```diff
@@ -122,3 +122,172 @@
+					flick("e_flash", user.flash)
+					switch(pick(1,2,3,4))
+						if(1)
+							user.dna.SetSEState(DEAFBLOCK,1)
+							user.dna.SetSEState(MUTEBLOCK,1)
+							user.dna.SetSEState(BLINDBLOCK,1)
+							genemutcheck(user,DEAFBLOCK,null,MUTCHK_FORCED)
+							genemutcheck(user,MUTEBLOCK,null,MUTCHK_FORCED)
+							genemutcheck(user,BLINDBLOCK,null,MUTCHK_FORCED)
+							user.update_mutations()
+							to_chat(user, "<span class=danger><B>You've gone blind, deaf and mute! </span></B>")
+						if(2)
+							for(var/datum/organ/external/l_arm/E in h.organs)
+								E.droplimb(1)
+							for(var/datum/organ/external/r_arm/E in h.organs)
+								E.droplimb(2)
+						if(3)
+							if(h.species.name != "Tajaran") //someone who was made a catbeast by the dice can't become a different species by getting lucky
+								switch(pick(1,2,3))
+									if(1)
+										if(h.species.name != "Unathi")
+											if(h.set_species("Unathi"))
+												h.regenerate_icons()
+											to_chat(user, "<span class=danger><B>You have been turned into a disgusting lizard! </span></B>")
+										else
+											for(var/datum/organ/external/l_arm/E in h.organs) //Someone who has already become a lizard can't get out of recieving a curse and so they lose their arms instead
+												E.droplimb(1)
+											for(var/datum/organ/external/r_arm/E in h.organs)
+												E.droplimb(1)
+									if(2)
+										if(h.species.name != "Skrell")
+											if(h.set_species("Skrell"))
+												h.regenerate_icons()
+											to_chat(user, "<span class=danger><B>You have been turned into a disgusting squidman! </span></B>")
+										else
+											for(var/datum/organ/external/l_arm/E in h.organs) //Someone who has already become a squid can't get out of recieving a curse and so they lose their arms instead
+												E.droplimb(1)
+											for(var/datum/organ/external/r_arm/E in h.organs)
+												E.droplimb(1)
+									if(3)
+										if(h.species.name != "Vox")
+											if(h.set_species("Vox"))
+												h.regenerate_icons()
+											to_chat(user, "<span class=danger><B>You have been turned into a dumb, diseased bird! </span></B>")
+										else
+											for(var/datum/organ/external/l_arm/E in h.organs) //Someone who is a vox can't get out of recieving a curse and so they lose their arms instead
+												E.droplimb(1)
+											for(var/datum/organ/external/r_arm/E in h.organs)
+												E.droplimb(1)
+						if(4)
+							h.adjustBrainLoss(200)
+							user.reagents.add_reagent("nutriment", 1000)
+							user.overeatduration = 1000
+
+							to_chat(user, "<span class=danger><B>In this moment you feel euphoric! </span></B>")
+				if(10 to 12)
+					to_chat(user, "<span class=sinister><B>You get nothing. No curse or reward! </span></B>")
+				if(13)
+					to_chat(user, "<span class=sinister><B>You've rolled 13! The cursed dice is broken! </span></B>")
+					explosion(get_turf(src), 0, 0, 4, 7)

```

Comment:
It was a side project of one of the coders about a year and a half ago, either dylan or nexis. It had some bugs and was completely dropped. Anyone is welcome to finish the system if they want.